projectGeneratorElectron/index.js

```diff
@@ -289,6 +300,27 @@ function parseAddonsAndUpdateSelect(arg) {
     console.log("in parseAddonsAndUpdateSelect " + arg);
     //path = require('path').resolve(__dirname, defaultOfPath + "/addons");
     addons = getDirectories(arg + "/addons","ofx");
+
+    var a = addons;
+    var b = addonsToSkip;
+    var addonsThatShouldBeShown = [];
+    var found;
+    for (var i = 0; i < a.length; i++) {
+        found = false;
+        // find a[i] in b
+        for (var j = 0; j < b.length; j++) {
+            if (a[i] == b[j]) {
+                found = true;
+                break;
+            }
+        }
+        if (!found) {
+            addonsThatShouldBeShown.push(a[i]);
+        }
+    }
+
+    addons = addonsThatShouldBeShown;
+
```

Comment:
oh, might be a later addition, i think `includes` instead of contains should work and this should work for sure:

```js
addons = addons.filter( function(addon) {
    return addonsToSkip.indexOf(addon)==-1;
});
```